packages/hermes/src/slack/sendSlack.ts

```diff
@@ -0,0 +1,41 @@
+import { WebClient } from "@slack/web-api";
+
+const slack = new WebClient(process.env.SLACK_TOKEN);
```

Comment:
Is there a specific reason why you are defining this variable before the next `import`? I think all `import` statement should be at the top of the file.

Reply:
Agreed, that shouldn't be in there. Will edit.